db/schema.rb

```diff
@@ -0,0 +1,231 @@
+# encoding: UTF-8
```

Comment:
Talan erdemesebb a schema.rb helyett SQL-t generalni. Foleg ha vannak olyan dolgok a semaban, amit a ruby-s DSL nem tud leirni. 

http://edgeguides.rubyonrails.org/active_record_migrations.html#types-of-schema-dumps